oktapam/constants/descriptions/resources.go

```diff
@@ -32,5 +32,6 @@ var (
 	ResourceResourceGroup                      = fmt.Sprintf("%s A PAM construct that contains a group of projects. %s", PrefixLEA, LinkResourceGroup)
 	ResourceResourceGroupProject               = fmt.Sprintf("%s A PAM construct that contains a collection of resources that share settings. %s", PrefixLEA, LinkResourceGroupProject)
 	ResourceResourceGroupServerEnrollmentToken = fmt.Sprintf("%s A token used to enroll servers in a PAM Project. %s", PrefixLEA, LinkResourceGroupServerEnrollmentToken)
+	ResourceSecretFolder                       = fmt.Sprintf("%s A file-system like construct which can contain other secret folders or secrets.", PrefixGA)
```

Comment:
```suggestion
	ResourceSecretFolder                       = fmt.Sprintf("%s A file-system like construct which contains other secret folders or secrets.", PrefixGA)
```

Reply:
Would something like this work instead?
```suggestion
	ResourceSecretFolder                       = fmt.Sprintf("%s A file-system like construct that contains secrets or nested secret folders.", PrefixGA)
```